build is sad:
```
======================================================================
FAIL: testIndexesSync (casexml.apps.phone.tests.test_sync_mode.MultiUserSyncTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 730, in testIndexesSync
    purge_restore_cache=True)
  File "corehq/ex-submodules/casexml/apps/case/tests/util.py", line 85, in assert_user_has_case
    return assert_user_has_cases(testcase, user, [case_id], return_single=True, **kwargs)
  File "corehq/ex-submodules/casexml/apps/case/tests/util.py", line 91, in assert_user_has_cases
    should_have=True, line_by_line=False, version=V2, **kwargs)
  File "corehq/ex-submodules/casexml/apps/case/tests/util.py", line 164, in check_user_has_case
    matches = [check_block(case_block) for case_block in case_blocks]
  File "corehq/ex-submodules/casexml/apps/case/tests/util.py", line 160, in check_block
    % (case_id, user.username, extra_info()))
AssertionError: Block for case_id 'indexes_sync_parent' doesn't appear in ota restore for user 'ferrel':
<ns0:case xmlns:ns0="http://commcarehq.org/case/transaction/v2" case_id="indexes_sync_parent" date_modified="2015-03-31" />
[]
```
I'll take a look probably tomorrow unless @snopoke beats me to it

--- a/corehq/ex-submodules/casexml/apps/phone/tests/performance_tests.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/performance_tests.py
@@ -137,7 +137,6 @@ class SyncPerformanceTest(SyncBaseTest):
                 version=V2,
                 index={'parent': (PARENT_TYPE, parent_case_id)}
             ).as_xml())
-            print("created child case", case_id)
         self._postFakeWithSyncToken(caseblocks, self.sync_log.get_id)
 
         referreal_cases = []
@@ -154,7 +153,6 @@ class SyncPerformanceTest(SyncBaseTest):
                 version=V2,
                 index={'parent': ('child', child_case_id)}
             ).as_xml())
-            print("created referral case", case_id)
         self._postFakeWithSyncToken(caseblocks, self.sync_log.get_id)
 
         all_cases = parent_cases + child_cases + referreal_cases

--- a/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py
@@ -72,8 +72,12 @@ class SyncBaseTest(TestCase):
         process_cases(form)
         return form
 
-    def _postFakeWithSyncToken(self, caseblock, token_id):
-        return post_case_blocks(list(caseblock), form_extras={"last_sync_token": token_id})
+    def _postFakeWithSyncToken(self, caseblocks, token_id):
+        if not isinstance(caseblocks, list):
+            # can't use list(caseblocks) since that returns children of the node
+            # http://lxml.de/tutorial.html#elements-are-lists
+            caseblocks = [caseblocks]
+        return post_case_blocks(caseblocks, form_extras={"last_sync_token": token_id})
 
     def _checkLists(self, l1, l2):
         self.assertEqual(len(l1), len(l2))

